wire-schema/build.gradle

```diff
@@ -14,7 +14,7 @@ dependencies {
   api project(':wire-runtime')
   api deps.kotlin.stdlib.jdk8
   implementation deps.okio.jvm
-  implementation deps.guava
```

Comment:
how many Guava usages remain?  shouldn't be many after konverting a bunch of classes to use Kotlin lists...

Reply:
Definitely not that many